benchmarks/util.py

```diff
@@ -50,7 +50,7 @@ def reset_rng_state(benchmark_experiment=None):
   torch.manual_seed(1337)
   random.seed(1337)
   np.random.seed(1337)
-  if benchmark_experiment is not None and benchmark_experiment.xla is not None:
+  if benchmark_experiment is not None and benchmark_experiment.xla is not None and not benchmark_experiment.use_torch_xla2:
```

Comment:
Don't we need to reset RNG state for `torch_xla2`?

Reply:
Not yet. How RNGs will be set in torch_xla2 is still under iteration. We can change once we figured out the API